addons/sale_timesheet/models/project.py

```diff
@@ -361,7 +361,7 @@ def _get_profitability_labels(self):
             'billable_time': _lt('Timesheets (Billed on Timesheets)'),
             'billable_milestones': _lt('Timesheets (Billed on Milestones)'),
             'billable_manual': _lt('Timesheets (Billed Manually)'),
-            'non_billable': _lt('Timesheets (Non Billable)'),
+            'non_billable': _lt('Timesheets (Non-Billable)'),
```

Comment:
Can you change your commit title and add the task reference into that commit `[IMP] sale_timesheet: Non Billable to Non-billable`?

```diff
- [IMP] sale_timesheet: Non Billable to Non-billable
+ [IMP] sale_timesheet: change `Non Billable` into `Non-billable`
```

Except if you squash all your commit? I don't know what you had in mind 🙂